internal/util/autoindex/function.go

```diff
@@ -0,0 +1,132 @@
+// Licensed to the LF AI & Data foundation under one
```

Comment:
check if this library will work, should be much easier to config
https://github.com/yidane/formula

Reply:
or might this library https://github.com/Sandertv/go-formula